### PR TITLE
test DF sizes #35

### PR DESCRIPTION
**Pull request by turbomam**
test DF sizes before merge #35

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "linkml-model-enrichment"
-version = "0.0.2"
+version = "0.0.3"
```
Comment (cmungall) [inline pyproject.toml:3]: I would do this in a separate PR, only immediately prior to release...

but no biggie for now, we don't even have releases for this repo yet

**Review by cmungall — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (turbomam) [reply pyproject.toml:3]: Yeah, that was an accidental mega merge. I have been asserting version because I'm pushing to PyPI now, for use in the NMDC DataHarmonizer. We should touch base on what method I use and what metadata you want to include in PyPI.
